config/environments/test.rb

```diff
@@ -22,14 +22,14 @@
     'Cache-Control' => "public, max-age=#{1.hour.to_i}",
   }
 
+  # Raise exceptions instead of rendering exception templates.
+  config.action_dispatch.show_exceptions = true
+
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false
```

Comment:
The application will render the public error pages instead of detailed debug information.